stock_valuation_account_manual_adjustment/wizards/mass_create_view.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record id="view_stock_valuation_account_mass_adjust"
+                model="ir.ui.view">
+            <field name="name">Mass Adjust Stock Valuation Account Discrepancies</field>
+            <field name="model">stock.valuation.account.mass.adjust</field>
+            <field name="type">form</field>
+            <field name="arch" type="xml">
+                 <form string="Reconcile Inventory Account">
+                     <group>
+                         <field name="increase_account_id"/>
+                         <field name="decrease_account_id"/>
+                         <field name="journal_id"/>
+                         <field name="remarks"/>
+                     </group>
+                     <newline/>
+                     <group colspan="2">
+                         <button name="process"
```

Comment:
You should put all buttons inside footer tag. 
Without footer tag:
![manual_valuation_wizard1](https://cloud.githubusercontent.com/assets/1947376/21272278/16f49ad0-c3f1-11e6-9137-b14ffef2d6e7.png)

With footer tag:
![manual_valuation_wizard2](https://cloud.githubusercontent.com/assets/1947376/21272288/22d28b28-c3f1-11e6-925b-807d8cec8eef.png)